Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayerSkia.cpp

```diff
@@ -91,6 +95,8 @@ Ref<Nicosia::Buffer> CoordinatedGraphicsLayer::paintTile(const IntRect& tileRect
 
     // Non-blocking, multi-threaded variant.
     if (auto* workerPool = m_coordinator->skiaUnacceleratedThreadedRenderingPool()) {
+        WTFBeginSignpost(this, PaintTile, "Skia unaccelerated multithread, dirty region %ix%i+%i+%i", tileRect.x(), tileRect.y(), tileRect.width(), tileRect.height());
+
```

Comment:
This is not the time to paint the tile, but the time to record the painting commands. Maybe we should rename it as RecordTile and add the Paint tile in the worker thread before and after the display list is replayed.

Reply:
Good catch.

Reply:
Done!